### PR TITLE
Veioss 400

### PR DESCRIPTION
**Pull request by modivilioglu**

### CODE REVIEW

File: app/pages/corrections/VatPeriodCorrectionsListPage.scala [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+import scala.concurrent.{ExecutionContext, Future}
+import scala.util.{Success, Try}
+
+case class VatPeriodCorrectionsListPage(period: Period, addAnother: Boolean) extends Page {
+ override def nextPageNormalMode(waypoints: Waypoints, answers: UserAnswers): Page = {
+        if(addAnother) {
+          answers.get(DeriveNumberOfCorrectionPeriods) match {
+            case Some(size) =>
+              ???
+            //Todo: Add when VEIOSS-371 merged
```
Comment (DevAndyroo) [inline app/pages/corrections/VatPeriodCorrectionsListPage.scala:36]: this has now been merged, so you should be able to implement this

File: test/pages/VatPeriodCorrectionsListPageSpec.scala [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    "must navigate in Normal mode" - {
+
+      "when the answer is yes" - {
+
+        "to CorrectionReturnPeriod when there are already some corrections" in {
+
+          val answers =
+            emptyUserAnswers
+              .set(CorrectionReturnPeriodPage(index), period).success.value
+//Todo: Add when VEIOSS-371 merged
```
Comment (DevAndyroo) [inline test/pages/VatPeriodCorrectionsListPageSpec.scala:43]: same here

**Review by DevAndyroo — CHANGES_REQUESTED** [review]

File: app/pages/corrections/VatPeriodCorrectionsListPage.scala (hunk 1 above)
Comment (modivilioglu) [reply app/pages/corrections/VatPeriodCorrectionsListPage.scala]: Done. Thanks

File: test/pages/VatPeriodCorrectionsListPageSpec.scala (hunk 2 above)
Comment (modivilioglu) [reply test/pages/VatPeriodCorrectionsListPageSpec.scala]: Done. Thanks

**Review by DevAndyroo — APPROVED** [review]

**Pull request merged**